PowerPointLabs/PowerPointLabs/Utils/GraphicsUtil.cs

```diff
@@ -191,6 +192,51 @@ private static void DeleteSpecificFilePath(string filePath)
         #endregion
 
         #region Slide
+        public static void ExportSlides(List<Slide> slides, string exportPath, float magnifyRatio = 1.0f)
+        {
+            if (slides.Count <= 0)
+            {
+                return;
+            } 
+            else if (slides.Count == 1)
+            {
+                ExportSlide(slides[0], exportPath, magnifyRatio);
+                return;
+            }
+
+            if (!Directory.Exists(exportPath))
```

Comment:
Change this into a guard clause? Seems like there is no code outside this if block.

Reply:
I refactored this to remove the if check.